client/src/ui/explore/OpportunityList.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link'; // Import Link for navigation
+// import { useRouter } from 'next/navigation'; // To handle URL params
+// import { SelectedFilters } from './Filters';
+import { OpportunityCard, OpportunityCardProps } from './OpportunityCard';
+import { createSupabaseClient } from '@/lib/supabase/client';
+import { Database } from '@/lib/types/database.types';
+
+interface OpportunityRaw {
+    company_name: string
+    created_at: string
+    opportunity_id: string
+    role_title: Database["public"]["Enums"]["OpportunityType"]
+    type: string
+    conversation: { conversation_id: string } | null;
+}
+
+const supabase = createSupabaseClient();
+
+// interface OpportunityListProps {
+//     filters: SelectedFilters;
+// }
+
+// TODO: Add filters to the OpportunityList component
+// export const OpportunityList: React.FC<OpportunityListProps> = ({ filters }) => {
+export const OpportunityList: React.FC = () => {
+
+  const [allOpportunities, setAllOpportunities] = useState<OpportunityCardProps[]>([]);
```

Comment:
I would prefer to have one source of truth for the applications that is fetched. Makes it easier to manage for us